app/Http/Middleware/LocalizationMiddleware.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+
+class LocalizationMiddleware
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle(Request $request, Closure $next)
+    {
+        $language = Session::get('lang', config('app.locate'));
+        $english = config('app.language.en');
+        $vietnamese = config('app.language.vi');
+        switch ($language) {
+            case $english:
+                $language = $english;
```

Comment:
```suggestion
                $language = config('app.language.en');
```
dùng như này, k cần thiết phải đặt thêm 1 biến như trên